script/env/install_env.sh

```diff
@@ -59,3 +59,6 @@ deployUser=${deployUser:-"dolphinscheduler"}
 
 # The root of zookeeper, for now DolphinScheduler default registry server is zookeeper.
 zkRoot=${zkRoot:-"/dolphinscheduler"}
+
+# whether the installation user has sudo permission
+sudoEnable=${sudoEnable:-"true"}
```

Comment:
Could you please update the docs about this? Thx

Reply:
@EricGao888  OK soon, Thanks